stdlib/public/Concurrency/AsyncStream.swift

```diff
@@ -191,30 +321,34 @@ public struct AsyncStream<Element> {
 
 @available(SwiftStdlib 5.5, *)
 extension AsyncStream: AsyncSequence {
-  /// The asynchronous iterator for iterating a AsyncStream.
+  /// The asynchronous iterator for iterating an asynchronous stream.
   ///
-  /// This type is specificially not Sendable. It is not intended to be used
-  /// from multiple concurrent contexts. Any such case that next is invoked
-  /// concurrently and contends with another call to next is a programmer error
-  /// and will fatalError.
+  /// This type is specificially not `Sendable`. Do not use it from multiple
```

Comment:
```suggestion
  /// This type specifically doesn't conform to `Sendable`. Do not use it from multiple
```

Spelling fix and avoid using type name as an adjective.